src/library/scala/collection/IterableOnce.scala

```diff
@@ -26,6 +26,9 @@ import scala.collection.mutable.StringBuilder
   *
   *    Note: will not terminate for infinite-sized collections.
   *
+  * @define willForceEvaluation
+  *    Note: Even when applied to a view or a lazy collection it will always force the elements.
```

Comment:
Also, I think that instead of “it will always force the elements” it might be clearer to say “this operation always evaluates the collection elements”